config/flags.go

```diff
@@ -361,6 +361,9 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.StringToString(TracingHeadersKey, map[string]string{}, "The headers to provide the trace indexer")
 
 	fs.String(ProcessContextFileKey, defaultProcessContextPath, "The path to write process context to (including PID, API URI, and staking address).")
+
+	// RPC Chain VMs
+	fs.Duration(RPCVMHandshakeTimeoutKey, constants.DefaultRPCVMHandshakeTimeout, "Maximum duration to wait for an RPC VM to complete its initialization handshake.")
```

Comment:
nit: it feels weird that the vm is called the `rpcchainvm` but we're referencing it as `rpc vm` here. My guess is that you wanted to avoid this since the name is long/awkward... I feel like something like `rpcchainvm-handshake-timeout` or `vm-grpc-handshake-timeout` might make sense but I don't have anything concretely in mind...